### ISSUE TITLE
_und_citation_name_map not generic to all flights/campaigns

### ISSUE THREAD

**Issue by swnesbitt** [nguy/Notebooks#1]
In `_und_citation_name_map`, it appears that the `name_map` was generated for the MC3E campaign, but the variable names change depending on each flight's calibration settings and project (GCPEx, IPHEx, OLYMPEX).  

For example

``` python
lwc1': "Liquid Water Content based on King Probe measurement "
                "adjusted (cloud threshold =  5.1 [#/cm^3], cloud interval "
                "= 30.0 [s] and adjustment slope = 0.500) for the baseline "
                "offset [g/m^3]",
```

This needs to be made more generic to search for only the beginning of the variable name, otherwise variables are being set to `None`.

I'll take a crack at this next week.

**Comment by nguy** [comment]
Yes, this has been a difficulty with airborne data (not just Citation). I used MC3E to generate a name_map, but following emails realize there are not standardizations. One thing I tried to do with readers was to allow the user to define a name_map dictionary. Unfortunately using a one-to-one transfer can lead to pretty difficult variable names (especially in Ames FFI format). I've also tried to allow some individual variable names downstream for working with the data.

I'm open to ideas on how to deal with this and am interested what you have in mind!
